doc/sphinx/language/gallina-extensions.rst

```diff
@@ -699,21 +698,22 @@ used by ``Function``. A more precise description is given below.
 
 .. cmdv:: Function @ident {* @binder } : @type := @term
 
-   Defines the not recursive function `ident` as if declared with `Definition`. Moreover
-   the following are defined:
+   Defines the not recursive function :token:`ident` as if declared with
+   :cmd:`Definition`. Moreover the following are defined:
 
-    + `ident_rect`, `ident_rec` and `ident_ind`, which reflect the pattern
-      matching structure of `term` (see :cmd:`Inductive`);
-    + The inductive `R_ident` corresponding to the graph of `ident` (silently);
-    + `ident_complete` and `ident_correct` which are inversion information
+    + :n:`@ident_rect`, :n:`@ident_rec` and :n:`@ident_ind`, which reflect the pattern
```

Comment:
@cpitclaudel This change is actually incorrect as I've just discovered because Sphinx thinks `ident_rect` is the token here. Can we get a way of indicating the token stops after `ident`? (For instance `:n:`@ident\_rect`.)

Reply:
Does ``` :token:`ident`\ ``_rect`` ``` work?

Reply:
It does but I was hoping we would try to avoid this...

Reply:
I agree, it's not ideal. Do we have token names with underscores in them? Otherwise underscores could be a separator.

Reply:
I think making the rendering uniform would be good.

Reply:
Unfortunately yes, we have many.

Reply:
>I think making the rendering uniform would be good.

If possible, I would make ``` :token:`foo` ``` look like ``` :n:`@foo` ```. But is that possible easily? It seems to me that this is something we might not have control over as it likely comes from the ReadTheDocs theme.

Reply:
It's easy to override parts of the theme, using our own CSS.  In `notations.css` you can change `.notation .hole {` into `.notation .hole, .std-token .pre {`; that should do it.

Reply:
For some strange reason I don't understand, your suggested change has the opposite effect (it uses the style of the grammar token - blue, non italic - everywhere). I don't really mind: this is pretty as well. But it feels weird that such change to the CSS would have such effect.

Reply:
Could it be that you broke the CSS, and thus that rule doesn't apply anymore? That change works for me when I live-test it in the firefox debugger :)

Reply:
Good guess. I had missed the comma.

Reply:
Fixed.